name.abuchen.portfolio/src/name/abuchen/portfolio/snapshot/PerformanceIndex.java

```diff
@@ -163,6 +163,13 @@ public double getFinalAccumulatedPercentage()
         return accumulated != null ? accumulated[accumulated.length - 1] : 0;
     }
 
+    public double getFinalAccumulatedAnnualizedPercentage()
+    {
+        double accumulatedPercentage = getFinalAccumulatedPercentage();
+        long days = getActualInterval().getDays();
+        return Math.pow(1 + accumulatedPercentage, 365d / ((double) days)) - 1;
```

Comment:
hmm... and 366 days? (Leap year)
i know... but the little tiny thinks... ;-)

Reply:
This code is part of #2321, not of this PR. See #2321